src/test/java/teammates/test/util/PriorityInterceptor.java

```diff
@@ -70,16 +70,6 @@ private String getClassName(IMethodInstance mi) {
             }
             
             private int packagePriorityOffset(String packageName) {
-                //Storage tests go first!
-                if (packageName.contains("storage")) {
-                    return 1000000;
-                }
-                //Action tests go last. (in component tests)
-                if (packageName.contains("teammates.test.cases.ui")) {
-                    return -1000000;
-                }
```

Comment:
These can be configured via the xml file itself (which has been done); configuring here only adds to maintenance burden.